src/rtl_433.c

```diff
@@ -1054,6 +1063,35 @@ static void parse_conf_option(r_cfg_t *cfg, int opt, char *arg)
             cfg->after_successful_events_flag = atobv(arg, 1);
         }
         break;
+    case 'u':
+#ifdef GPSD
+        if (arg && strlen(arg) != 1) {
+            //arg = arg + 1;
+            int i, count;
+            for (i = 0, count=0; arg[i]; i++) {
+              count += (arg[i] == ':');
+            }
+            if(count != 1) {
+                (void)fprintf(stderr,"GPS requires a host and port.\n");
+                usage(1);
+            }
```

Comment:
How about to default to localhost and port 2947 if no arguments are given?

Reply:
I agree that defaulting to localhost:2947 is good.   My guess at the normal use case is a laptop that is moving, so having gpsd on the same laptop is the obvious approach.